run_scripts/3_generate_outputs.sh

```diff
@@ -32,7 +32,11 @@ GOOGLE_CLOUD_CREDENTIALS_FILE_PATH=$2
 PIPELINE_CONFIGURATION_FILE_PATH=$3
 DATA_ROOT=$4
 
-mkdir -p "$DATA_ROOT/Coded Coda Files"
+if [ -d "$DATA_ROOT/Outputs" ]
```

Comment:
Please add a comment explaining why this directory needs to be deleted

Reply:
updated thanks.